CoreFoundation/Stream.subproj/CFSocketStream.c

```diff
@@ -218,9 +218,9 @@ CF_PRIVATE CFStreamError _CFStreamErrorFromError(CFErrorRef error) {
 }
 
 CF_PRIVATE CFErrorRef _CFErrorFromStreamError(CFAllocatorRef alloc, CFStreamError *streamError) {
-    CFErrorRef result;
+    CFStreamError result;
```

Comment:
This is not correct. I'd be surprised if it compiles. Please revert to the original.

Reply:
Another case and point on my newest comment.